streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -493,6 +493,7 @@ protected void maybeClean() {
      * Commit the states of all its tasks
      */
     private void commitAll() {
+        log.trace("stream-thread [{}] commiting all tasks", this.getName());
```

Comment:
To be consistent with other logging entries: `... Committing all its owned tasks`. And capitalize on line 509 below as well.